rero/rero-ils
# Libraries: Form Validation

* NEW: Library Form Validation

How to test:
Run the scripts `bootstrap` and `setup`
Check form validation on edit.
Fields Required:
- Name
- Code (test unique valuer)
- Time Range (start_time > end_time)
- Date Exception: title, Time period, etc…

Co-authored-by: Peter Weber <peter.weber@rero.ch>
Signed-off-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
